File: New/Hacker Rank/Assignment 1/py/Scripts/Mini-Max Sum.py
#Mini-Max Sum
def miniMaxSum(arr,n):
    max=arr[0]
    min=arr[0]
    maxsum=0
    minsum=0
    for i in range(n):
        if max<arr[i]:
            max=arr[i]
        if min>arr[i]:
            min=arr[i]
    if max == min:
        n=n-1
    for i in range(n):
        if arr[i]!=min:
            maxsum=maxsum+arr[i]
        if arr[i]!=max:
            minsum=minsum+arr[i]
        if max==min:
            maxsum=maxsum+arr[i]
            minsum=maxsum
    print(minsum,maxsum)

arr=list()
a=5
for k in range(a):
    arr.append(int(input()))
miniMaxSum(arr,a)

File: New/Hacker Rank/Assignment 1/cpp/Mini-Max Sum.cpp
//Mini-Max Sum
#include<iostream>
using namespace std;
void miniMaxSum(long int arr[],int a)
{
    long int maximum=arr[0],minimum=arr[0];
    long int minsum=0,maxsum=0;
    for(int i=0;i<a;i++)
    {
        if(arr[i]<minimum)
        {
            minimum=arr[i];
        }
        if(arr[i]>maximum)
        {
            maximum=arr[i];
        }
    }
    if(maximum==minimum){a--;}
    for(int i=0;i<a;i++)
    {
        if(arr[i]!=maximum)
        {
            minsum=minsum+arr[i];
        }
        if(arr[i]!=minimum)
        {
            maxsum=maxsum+arr[i];
        }
        if(maximum==minimum)
        {
            maxsum=maxsum+arr[i];
            minsum=maxsum;
        }
    }
    cout<<minsum<<" "<<maxsum;
}
int main()
{
    long int arr[5];
    for(int i=0;i<5;i++)
    {
        cin>>arr[i];
    }
    miniMaxSum(arr,5);
    return 0;
}
